src/v2/Components/Ribbon.tsx

```diff
@@ -0,0 +1,85 @@
+import { Flex, Text } from "@artsy/palette"
+import * as React from "react"
+import { Media } from "v2/Utils/Responsive"
+
+interface RibbonProps {
+  ribbonText: string
+  ribbonDivider?: string
+}
+
+export const Ribbon: React.FC<RibbonProps> = ({
+  ribbonText,
+  ribbonDivider = "•",
+}) => {
+  return (
+    <>
+      <Media lessThan="sm">
+        <Flex
+          bg="black100"
+          justifyContent="space-between"
+          alignItems="center"
+          px={0.5}
+          height={20}
+        >
+          <Text variant="md" color="white">
+            {ribbonDivider}
+          </Text>
+          <Text variant="xs" color="white">
+            {ribbonText}
+          </Text>
+          <Text variant="md" color="white">
+            {ribbonDivider}
+          </Text>
+          <Text variant="xs" color="white">
+            {ribbonText}
+          </Text>
+          <Text variant="md" color="white">
+            {ribbonDivider}
+          </Text>
+        </Flex>
+      </Media>
+      <Media greaterThanOrEqual="sm">
+        <Flex
+          bg="black100"
+          justifyContent="space-between"
+          alignItems="center"
+          height={20}
+        >
+          <Text variant="xs" color="white">
+            {ribbonText}
+          </Text>
+          <Text variant="md" color="white">
+            {ribbonDivider}
+          </Text>
+          <Text variant="xs" color="white">
+            {ribbonText}
+          </Text>
+          <Text variant="md" color="white">
+            {ribbonDivider}
+          </Text>
+          <Text variant="xs" color="white">
+            {ribbonText}
+          </Text>
+          <Text variant="md" color="white">
+            {ribbonDivider}
+          </Text>
+          <Text variant="xs" color="white">
+            {ribbonText}
+          </Text>
+          <Text variant="md" color="white">
+            {ribbonDivider}
+          </Text>
+          <Text variant="xs" color="white">
+            {ribbonText}
+          </Text>
+          <Text variant="md" color="white">
+            {ribbonDivider}
+          </Text>
+          <Text variant="xs" color="white">
+            {ribbonText}
+          </Text>
+        </Flex>
+      </Media>
+    </>
+  )
```

Comment:
Yeah, we can get rid of both the replication and the media component usage. Along with reducing the number of DOM nodes. If you want we could pair on an implementation in Palette?